patches/minecraft/net/minecraft/network/play/server/SCommandListPacket.java.patch

```diff
@@ -1,12 +0,0 @@
---- a/net/minecraft/network/play/server/SCommandListPacket.java
-+++ b/net/minecraft/network/play/server/SCommandListPacket.java
-@@ -119,6 +119,9 @@
-          String s = p_197695_0_.func_150789_c(32767);
-          ArgumentType<?> argumenttype = ArgumentTypes.func_197486_a(p_197695_0_);
-          if (argumenttype == null) {
-+            if ((i & 16) != 0) { // FORGE: Flush unused suggestion data
```

Comment:
This must of changed in a MC update. It needs to be `buf & 16` not `i & 16`to match the case below it.

Reply:
I know that `buf & 16` was the original intention. However the ResourceLocation this patch is trying to read is not the next thing to read here. If `argumenttype` is `null` then the `ArgumentType` was not found, so whatever data it wrote is next in the buffer. _After_ that data the custom suggestion `ResourceLocation` is to be read. The patch was added by @tterrag1098 back in 2019, where it correctly checked `buf & 16`. However I am unsure how that is supposed to have ever fixed anything - for the reasons outlined above.